docs/enums/tableau.analyticsobjecttype.html

```diff
@@ -87,30 +87,30 @@ <h2>Enumeration members</h2>
 				<section class="tsd-panel tsd-member tsd-kind-enum-member tsd-parent-kind-enum">
 					<a name="cluster" class="tsd-anchor"></a>
 					<h3>Cluster</h3>
-					<div class="tsd-signature tsd-kind-icon">Cluster<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-symbol"> =&nbsp;&quot;cluster&quot;</span></div>
+					<div class="tsd-signature tsd-kind-icon">Cluster<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-symbol"> = &quot;cluster&quot;</span></div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in Namespaces/Tableau.ts:26</li>
+							<li>Defined in src/ExternalContract/Namespaces/Tableau.ts:26</li>
```

Comment:
@nikhillakshman @johnDance   Does it matter if we show the src path in the reference page?  Shouldn't we just show `Namespaces/Tableau.ts` ?

Reply:
If you don't know where this came from either, it's probably a configuration change in docs generation-
Do either of you know? john, nikhil

Reply:
Looks like it picks up that path in the Typdoc build command.  
From package.json: 
`--out docs-extensions src/ExternalContract`  

I think `docs-extensions` is the name of the directory the HTML files get dumped in. Not sure about `/scr/ExternalContract`  - could be the path to files to process.

Reply:
TypeDoc now adds the path from the root in the "Defined in" statement.